work/worker.go

```diff
@@ -750,25 +779,43 @@ CommitTransactionLoop:
 		// Start executing the transaction
 		env.state.SetTxContext(tx.Hash(), common.Hash{}, env.tcount)
 
-		err, logs := env.commitTransaction(tx, bc, rewardbase, vmConfig)
+		// Verify that tx is included in the bundle
+		targetBundle := &builder.Bundle{}
+		numShift := 1
+		if bundleIdx := builder_impl.FindBundleIdx(bundles, tx); bundleIdx != -1 {
```

Comment:
I think it can't detect the first lend tx since lend tx is just created by `txGen` so isn't in the `bundle.BundleTxs`.

Reply:
Oh, that's true.
Has, which is used when creating BFS, is not expected to include TxGenerator, so it seems necessary to separate this.

Reply:
It was fixed at https://github.com/kaiachain/kaia/commit/c866a31888141d2d51e538da6e2285b29d8f8150. Thank you.